src/OrchardCore.Themes/TheTheme/Views/Layout.cshtml

```diff
@@ -1,28 +1,39 @@
+@using OrchardCore.Admin.Models
 @using OrchardCore.DisplayManagement
 @using OrchardCore.DisplayManagement.ModelBinding
+@using OrchardCore.Entities
+@using OrchardCore.Themes.Services
 @using OrchardCore.Users.Models
 
 @inject IDisplayManager<UserMenu> DisplayManager
 @inject IUpdateModelAccessor UpdateModelAccessor
-
+@inject ThemeTogglerService ThemeTogglerService
+@{
+    var adminSettings = Site.As<AdminSettings>();
+}
 <!DOCTYPE html>
-<html lang="@Orchard.CultureName()">
+<html lang="@Orchard.CultureName()" dir="@Orchard.CultureDir()" data-bs-theme="@await ThemeTogglerService.CurrentTheme()" data-tenant="@ThemeTogglerService.CurrentTenant">
 <head>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>@RenderTitleSegments(Site.SiteName, "before")</title>
-    <link type="image/x-icon" rel="shortcut icon" href="~/TheTheme/favicon.ico">
+    <link type="image/x-icon" rel="shortcut icon" href="~/TheTheme/images/favicon.ico">
+
+    <!-- This script can't wait till the footer -->
+    <script asp-name="admin-head" version="1" at="Head"></script>
```

Comment:
Why would TheTheme need `admin` scripts?

Reply:
@agriffard 
The admin-head scripts contains the same code that we use for the toggler and setting theme preferences.

The good thing is that we use the same script once and we cache and reuse it

Reply:
Ok but there is no need for the `admin-menu` then, right?

Reply:
No admin-menu is for the admin menu